app/src/main/java/com/x8bit/bitwarden/data/platform/datasource/network/model/ConfigResponseJson.kt

```diff
@@ -31,7 +31,7 @@ data class ConfigResponseJson(
     val environment: EnvironmentJson?,
 
     @SerialName("featureStates")
-    val featureStates: Map<String, Boolean>?,
+    val featureStates: Map<String, String>?,
```

Comment:
I think it might make more sense to parse this as a `JsonElement` or `JsonPrimitive`, I don't think we would need to make the `Json` instance `isLenient` and it would imply the unsafe nature of this property.

Reply:
let me try that out.

Reply:
It worked out.

Reply:
Awesome!